aleph-client/src/staking.rs

```diff
@@ -112,6 +112,21 @@ pub fn wait_for_next_era<C: AnyConnection>(connection: &C) -> anyhow::Result<Era
     wait_for_era_completion(connection, get_current_era(connection) + 1)
 }
 
+pub fn wait_for_at_least_era<C: AnyConnectionExt>(
+    connection: &C,
+    era: EraIndex,
+) -> anyhow::Result<EraIndex> {
+    let sessions_per_era: u32 = connection.read_constant(PALLET, "SessionsPerEra");
+    let mut current_era = get_era(connection, None);
+    while current_era < era {
```

Comment:
why do we wait in loop? `wait_for_session` doesn't have any timeout (unfortunately) so it should be enough to just `wait_for_session(connection, era * sessions_per_era + 1)`

Reply:
Removed. That was part of some other effort to allow e2e-tests run one after another (some of them are messing others assumptions) and possibly run them on historical blocks (not wait for new sessions but execute tests for some given period - it could be a lot faster). But I've abandoned hope that it is doable.